app/components/m-header.js

```diff
@@ -26,8 +26,7 @@ export default class MHeader extends Component {
       this.environmentClass = 'vlc-environment-pill--test';
     }
 
-    if (window.location.href.indexOf('https://kaleidos.vlaanderen.be') === 0
-      && this.currentSession.checkIsDeveloper()) {
+    if (window.location.href.indexOf('https://kaleidos.vlaanderen.be') === 0) {
```

Comment:
Deze willen we waarschijnlijk in z'n geheel weg ... de eindgebruikers hebben weinig boodschap aan de pill `PROD` (?). +
- if -> else if
- window.location.hostname met straight `===` ipv `indexOf` ...

Reply:
Correct. 
Ik heb de logica nog wat verder opgekuist om op hostname te checken en met config-map te werken ipv `if-else`.